### PR TITLE
changed strategy of which Qt binding to use, and removed sitePpackage…

### PR DESCRIPTION
**Pull request by sanfx**
Hi, 

I have made few changes thought you might like it, I have changed the strategy you set which qt bindings you want to use, I also think the (just suggestions)

1. configuration settings shouldn't be done inside this boilerplate module, rather should be done in ini or config ascii file
2. Instead of using print you should use logger.

for the above two I will wait for what you want to go with then I will put more time to set it up.

Cheers,
san

### CODE REVIEW

File: boilerplate.py [hunk 1]
```diff
@@ -30,15 +30,6 @@
 launchAsPanel = False										# False = opens as regular window, True = opens as panel
 parentToNukeMainWindow = True									# True = makes window stay on top of Nuke
 
-# Site-packages location:
-site_packages_Win = ''										# Location of site-packages containing PySide and pysideuic and/or PyQt and SIP
-site_packages_Linux = ''									# Location of site-packages containing PySide and pysideuic and/or PyQt and SIP
-site_packages_OSX = ''										# Location of site-packages containing PySide and pysideuic and/or PyQt and SIP
-#site_packages_Win = 'C:/Python26/Lib/site-packages'						# Example: Windows 7
-#site_packages_Linux = '/usr/lib/python2.6/site-packages'					# Example: Linux CentOS 6.4
-#site_packages_OSX = '/Library/Python/2.7/site-packages'					# Example: Mac OS X 10.8 Mountain Lion
```
Comment (sanfx) [inline boilerplate.py]: should be done in shell/terminal or where PATH is set in windows not here.

File: boilerplate.py [hunk 2]
```diff
@@ -48,32 +39,27 @@
...
-except:
+except ImportError:
 	pass
 
 
-''' PySide or PyQt '''
-''' --------------------------------------------------------------------------------------------------------------------------------------------------------- '''
-if (site_packages_Win != '') and ('win' in sys.platform): sys.path.append( site_packages_Win )
-if (site_packages_Linux != '') and ('linux' in sys.platform): sys.path.append( site_packages_Linux )
-if (site_packages_OSX != '') and ('darwin' in sys.platform): sys.path.append( site_packages_OSX )
```
Comment (sanfx) [inline boilerplate.py]: if you don't even set the site_packages_xxx variable in this file and you already have PyQt path included in the python path, you are unnecessary creating a variable and setting an empty string.

File: boilerplate.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
 ''' --------------------------------------------------------------------------------------------------------------------------------------------------------- '''
 
 # General
-QtType = 'PySide'										# Edit this to switch between PySide and PyQt
+_qtBindings = 'PySide'										# Edit this to switch between PySide and PyQt
```
Comment (sanfx) [inline boilerplate.py]: changed variable name to protected _qtBindings as QtType can be misleading, having it protected ensures its not accessible directly outside the module scope.

File: boilerplate.py (hunk 1 above)
Comment (fredrikaverpil) [inline boilerplate.py]: I don't completely agree with that statement, as there are reasons for why you may want to control which version of PySide or PyQt you want to use – and this allows you to control this from within the boilerplate.

However, I agree that the current setup is clumsy. I will accept your pull request but I will also make an ammendment, which allows the user to set a path which will get added to sys.path – if specified.

**Pull request merged**